Pages/Student/Create.cshtml

```diff
@@ -1,47 +1,117 @@
 @page
 @model DB_College_Management.Pages.Student.CreateModel
 
+<html>
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600" rel="stylesheet" type="text/css">
+ <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+<style>
+ body{
+     background-image: url(https://banner2.cleanpng.com/20180326/lje/kisspng-polygon-geometry-plane-desktop-wallpaper-abstract-5ab9c141484b91.1154809315221230732961.jpg);
+     background-repeat: no-repeat;
+    background-size:100%;
+     
```

Comment:
You cannot add custom css here as this a template not an html page. Please use only bootstrap classes for styling.